core/crypto/CMakeLists.txt

```diff
@@ -3,6 +3,19 @@
 # SPDX-License-Identifier: Apache-2.0
 #
 
+add_library(crypto
+        hasher.hpp
+        hasher/hasher_impl.hpp
+        hasher/hasher_impl.cpp
+        )
+
 add_subdirectory(blake2)
 add_subdirectory(twox)
 add_subdirectory(sha)
+
+target_link_libraries(crypto
```

Comment:
you should write `target_link_libraries` right after `add_library`

Reply:
Fixed.